outofplace_1801_bronze.java

```diff
@@ -0,0 +1,43 @@
+/*
+Bill Zheng
+Time: 36 min
+USACO Test Cases: **********
+1-10 Difficulty: 6, need to find the right solution
+Reflection: Initially I forgot how to do this problem and so I was trying to work a super long method of swapping cows and checking conditions
+	to see if the cow was in the right place. I did this for about 30 min until I realized it would be way to hard to check all of the edge
+	cases. And so, after 30 min I decided to look back at some code I did in the past and I found the easiest method to solve the problem
+	by comparing two arrays, one sorted and one not sorted. The code for the final solution took like 2 minutes to write. This entire problem
+	was just me digging into a dead end and then looking back to find the correct and easiest solution.

```

Comment:
Clever solution is nice, but it is also a win-it-all/lose-it-all deal. You need practice on logics that your *intuition* can relate to so you can reliably produce results.